build.gradle

```diff
@@ -10,34 +10,32 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.8.4'
-        classpath 'com.gradle.publish:plugin-publish-plugin:0.10.1'
         classpath 'com.netflix.nebula:nebula-publishing-plugin:12.0.0'
-        classpath 'com.palantir.baseline:gradle-baseline-java:0.62.1'
-        classpath 'com.palantir.gradle.consistentversions:gradle-consistent-versions:1.9.2'
-        classpath 'gradle.plugin.com.palantir.gradle.gitversion:gradle-git-version:0.7.3'
-        classpath 'gradle.plugin.org.inferred:gradle-processors:2.2.0'
     }
 }
 
+plugins {
```

Comment:
could we do this separately?

Reply:
don't think it's worth creating a new PR for this